app/views/layouts/_guide.html.erb

```diff
@@ -32,7 +32,8 @@
   </div>
 </div>
 
-<h3><%= t :content %></h3>
+<h3 style="display:inline;"><%= t :content %></h3>
+<%= restart_guide_link(@guide) if current_user && @guide.started?(current_user) %>
```

Comment:
It would be nice to have a view spec, that tests he following scenarios: 

* The trash button is shown when there is progress
* The trash button is not shown, otherwise
* A user can click the trash button and the progress is cleared.